scheduler/test/cook/test/kubernetes/api.clj

```diff
@@ -13,47 +13,58 @@
 
 (deftest test-get-consumption
   (testing "correctly computes consumption for a single pod"
-    (let [pods [(tu/pod-helper "podA" "hostA" {:cpus 1.0 :mem 100.0})]
+
+    (let [pods [(tu/pod-helper "podA" "hostA" {:cpus 1.0 :mem 100.0 :gpus "2"})]
```

Comment:
Do we still have tests for 0 gpu?

Reply:
in this same function, within `(testing "correctly aggregates pods by node name")`, podD on hostC is an example of how a pod with 0 gpus is treated (the `get-consumption` function returns `:gpus {}` in that case). I have added in another test that solely has 0 gpus, though